modules/knative-eventing-CR-system-deployments.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-configuration.adoc
+
+:_content-type: REFERENCE
+[id="knative-eventing-CR-system-deployments_{context}"]
+= Overriding Knative Eventing system deployment configurations
+
+You can override the default configurations for some specific deployments by modifying the `deployments` spec in the `KnativeEventing` custom resource (CR). Currently, overriding default configuration settings is supported for the `eventing-controller`, `eventing-webhook`, and `imc-controller` fields.
+
+In the following example, the `KnativeEventing` CR overrides the `eventing-controller` deployment to request 0.3 CPU and 100MB of RAM for the `eventing-controller` container, and sets hard limits of 1 CPU and 250MB RAM:
+
+.KnativeEventing CR example
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  deployments:
+  - name: eventing-controller
+    resources:
+    - container: eventing-controller
+      requests:
+        cpu: 300m
+        memory: 100Mi
+      limits:
+        cpu: 1000m
+        memory: 250Mi
+----
```

Comment:
MiB and MB are a bit different [1], 100Mi is 100 Mebibytes, while 100MB should be 100M (100 Megabytes), so we either change the description to use `MiB` instead of `MB` or we change the memory setting to use `M` instead of `Mi` [2].

[1] https://www.google.com/search?q=Mi+to+MB&oq=Mi+to+MB&aqs=chrome..69i57j0i512j0i67j0i512l2j69i60j69i61l2.4103j0j7&sourceid=chrome&ie=UTF-8
[2] https://stackoverflow.com/questions/50804915/kubernetes-size-definitions-whats-the-difference-of-gi-and-g

Reply:
it's a small comment but the PR looks good to me.

Reply:
Perfect, thanks for letting me know! This came from upstream so I guess we also need to fix it in the docs there 😉